src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -64,8 +63,8 @@
   private final KeyUtil keyUtil;
 
   Button reloadKeyBtn = new Button();
+  final Button disconnectBtn = new Button();
 
-  private Pane emptyPane = new Pane();
```

Comment:
It seems a bit weird to me, that the pane is "lost"... Don't we need it to ensure a good looking gui? @JenniferMue & @martinfrancois